### PR TITLE
Update Majordomo.ru.xml

### PR DESCRIPTION
**Pull request by ivysrono**

### CODE REVIEW

File: src/chrome/content/rules/Majordomo.ru.xml [hunk 1]
```diff
@@ -1,59 +1,12 @@
...
-	<!--	Direct rewrites:
-				-->
+<ruleset name="Majordomo.ru">
 	<target host="majordomo.ru" />
-	<!--target host="book.majordomo.ru" /-->
-	<target host="leto.majordomo.ru" />
-	<target host="old.majordomo.ru" />
 	<target host="www.majordomo.ru" />
+	<target host="book.majordomo.ru" />
+	<target host="old.majordomo.ru" />
```
Comment (J0WI) [inline src/chrome/content/rules/Majordomo.ru.xml]: Could you also include the other subdomains? There are some more.

**Comment by ivysrono** [comment]
`ERROR src/chrome/content/rules/Majordomo.ru.xml: Fetch error: http://billing2.majordomo.ru/ => https://billing2.majordomo.ru/: (6, 'Could not resolve host: billing2.intr')`

Why？
@J0WI @Hainish

**Comment by J0WI** [comment]
```
$ curl -IL https://billing2.majordomo.ru/
HTTP/1.1 301 Moved Permanently
Server: nginx
Date: Sun, 16 Apr 2017 16:26:43 GMT
Content-Type: text/html
Content-Length: 178
Connection: keep-alive
Location: https://billing2.intr/

curl: (6) Could not resolve host: billing2.intr
```

**Comment by ivysrono** [comment]
@J0WI Ok now?

**Pull request merged**